src/references/referenceController.ts

```diff
@@ -0,0 +1,19 @@
+import {
+  Body,
+  Controller,
+  Post,
+  Route,
+  SuccessResponse,
+} from "tsoa";
+import { IFormattedText } from "./reference";
+import { referenceService } from "./referenceService";
+
+@Route("formatter")
+export class referenceController extends Controller {
```

Comment:
Class names should be [PascalCase](https://techterms.com/definition/pascalcase)

```suggestion
export class ReferenceController extends Controller {
```